frontend/src/components/buttons/EditCancelButton.tsx

```diff
@@ -0,0 +1,36 @@
+import { Button } from "primereact/button";
+
```

Comment:
Rename file include the isAddPage functionality maybe EditCancelButtonforAddDetailPage